api/shipping_categories.rst

```diff
@@ -0,0 +1,424 @@
+Shipping Categories
+===================
+
+These endpoints will allows you to easily manage shipping categories. Endpoints' base is `/api/shipping-categories`.
+
+Shipping categories have two different serialization groups. While presenting a list you will receive an object with following fields:
+
++-------+-------------------------------------+
+| Field | Description                         |
++=======+=====================================+
+| id    | Id of shipping category             |
++-------+-------------------------------------+
+| name  | Name of shipping category           |
++-------+-------------------------------------+
+| code  | Unique shipping category identifier |
++-------+-------------------------------------+
+
+If you request for a more detailed data, you will receive an object with following fields:
```

Comment:
If you request a single resource, ``Detailed`` serialization group will be used and following fields exposed:

Detailed vs. Details? Or maybe even, General and Detailed? Thoughts?